packages/server/cloud/nlp/package.json

```diff
@@ -13,24 +12,28 @@
     "start": "functions-framework --source=build/src/ --target=nlp --port=7869",
     "watch": "concurrently \"tsc -w\" \"nodemon --watch ./build/ --exec npm run start\"",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "gts lint",
-    "fix": "gts fix",
-    "clean": "gts clean",
-    "compile": "npm run lint && npm run clean && tsc",
-    "deploy": "npm run compile && gcloud functions deploy nlp --entry-point nlp --project adaptive-trader --trigger-http --allow-unauthenticated --runtime nodejs14"
+    "format:check": "prettier . --check",
+    "lint": "eslint src --max-warnings 0",
+    "lint:fix": "eslint src --max-warnings 0 --fix",
+    "build": "tsc",
+    "deploy": "gcloud functions deploy nlp2 --gen2 --region=us-central1 --entry-point nlp --project adaptive-trader --runtime nodejs20 --trigger-http --allow-unauthenticated"
```

Comment:
### Important
.. deploying to gen2 function called `nlp2`
.. update to current LTS runtime, and set region as demanded now on Gen 2, rest is the same
.. Google `dependencies` fully upgraded, all seems fine